package.json

```diff
@@ -121,7 +121,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.6.0",
-    "fundamental-styles": "0.19.0-rc.43",
+    "fundamental-styles": "0.20.0-rc.15",
```

Comment:
Not sure we're allowed to bump deps versions in regular tickets. We have @dependabot tickets for that and until we blocked for example due to the fund-styles version we apply `blocked` label.

Reply:
Okay if there is no breaking changes.